CMakeLists.txt

```diff
@@ -82,6 +84,14 @@ add_custom_target(
             ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}
 )
 
+add_custom_target(
```

Comment:
I don't think we should add a general custom target for something which is very specific to only a few samples.

Reply:
change CopyYAML to something specific, and treat it specific outside of loop